tools/ci/check_grep2.py

```diff
@@ -151,6 +151,11 @@ def check_href_styles(idx, line):
     if HREF_OLD_STYLE.search(line):
         return [(idx + 1, "BYOND requires internal href links to begin with \"byond://\"")]
 
+EMPTY_LIST_WHITESPACE = re.compile(r"list\(\s+")
```

Comment:
It seems that whitespace counts newlines, whoops
```suggestion
EMPTY_LIST_WHITESPACE = re.compile(r"list\(\s+/)")
```

Reply:
Made the regex catch nearly issue: https://regex101.com/r/WCCmUf/1

PTAL